doc/classes/ProjectSettings.xml

```diff
@@ -917,6 +917,7 @@
 		</member>
 		<member name="editor/export/convert_text_resources_to_binary" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], text resources are converted to a binary format on export. This decreases file sizes and speeds up loading slightly.
+			[b]Note:[/b] Project Settings are not a resource so "project.godot" will always be converted to "project.binary" on export.
 			[b]Note:[/b] If [member editor/export/convert_text_resources_to_binary] is [code]true[/code], [method @GDScript.load] will not be able to return the converted files in an exported project. Some file paths within the exported PCK will also change, such as [code]project.godot[/code] becoming [code]project.binary[/code]. If you rely on run-time loading of files present within the PCK, set [member editor/export/convert_text_resources_to_binary] to [code]false[/code].
```

Comment:
This needs to be changed to some useful example, it now contradicts what it just said above

Reply:
I think the converted name (`export-{md5}-foo.scn`) is an implementation detail, no?

Reply:
Agreed, so let's just mention it like "it will turn `tscn` to `scn` and `tres` to `res`" and perhaps mention something about how you can't open these with `FileAccess`, as per:
* https://github.com/godotengine/godot/issues/87494

And so on, unsure exactly where to refer to